Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -743,7 +744,23 @@
                         .withId("note")
                         .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
                         .withContentOverflow()
-                        .withSourceValueEchoRendering());
+                        .withSourceValueEchoRendering())
+                .withColumn(builder =>
```

Comment:
you added the "validaccordingtolifecycle" to the read model but is not using it... lets add a column next to the other "validity column"

"Aktiv ifølge status"